var/spack/repos/builtin/packages/conquest/package.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Conquest(MakefilePackage):
+    """CONQUEST is a DFT code designed for large-scale calculations,
+    with excellent parallelisation."""
+
+    homepage = "http://www.order-n.org/"
+    url = "https://github.com/OrderN/CONQUEST-release/releases/download/v1.2/CONQUEST-release-1.2.tar.gz"
+    git = "https://github.com/OrderN/CONQUEST-release/"
+
+    # notify when the package is updated.
+    maintainers("davidbowler", "tkoskela", "ilectra")
+
+    version("1.2", sha256="74d974f20ec15ff31d97cd42aae6dbe95288eedfa785896d5872b9ff44ee7ae2")
+    version("1.1", sha256="772e058f073cccfee45521aa62bb13192ab07cb2979b6076ddbf21ba22f9ba5d")
+    version("master", branch="master")
+    version("develop", branch="develop")
+
+    depends_on("blas")
+    depends_on("lapack")
+    depends_on("scalapack")
+    depends_on("fftw-api")
+    depends_on("libxc@:5")
+    depends_on("mpi")
+
+    variant("openmp", default=False, description="Build with OpenMP support")
+    variant("mult_kern", default="default", description="Matrix multiplication kernel type")
```

Comment:
Can you add a list of allowed values?

Reply:
Done. I also had to fix a bug with the OpenMP variant, sorry about the extra change